openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -122,8 +122,9 @@ Both the Wallet and the Credential Issuer:
 
 * MUST support the authorization code flow.
 * MUST support protocol extensions for the SD-JWT VC Credential format profile as defined in (#vc_sd_jwt_profile).
-* MUST support sender-constrained tokens using the mechanism defined in [@!RFC9449].
-* MUST support [@!RFC7636] with `S256` as the code challenge method.
+* MUST support sender-constrained tokens using DPoP as defined in [@!RFC9449].
+* MUST comply with the [@!FAPI2_Security_Profile], including but not limited to using PKCE [@!RFC7636] with `S256` as the code challenge method, Pushed Authorization Requests (PAR) [@!RFC9126] and the `iss` value in the Authorization response [@!RFC9207]. The following exception to [@!FAPI2_Security_Profile] applies:
```

Comment:
I would list all mechanisms and extensions explicitly and then they implementers should respect security guidance by FAPI 2.0 security profile, **were applicable.**

Reply:
I don't think it makes sense to try and restate the FAPI2 requirements in HAIP; restating normative text from other specifications normatively can be confusing, and it is easy to say things that contradict the other standard(s).